packages/core-js/test/utils/logger.test.js

```diff
@@ -61,7 +61,7 @@ describe('utils/logger.js', () => {
           '┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓',
         ],
         [
-          '┃ ‣ hello........................................................................................... ┃',
```

Comment:
Removed these lines in the info box (also affects the deployer) because it doesn't allow you to alt-click to select an address, which is annoying.